kiwi/defaults.py

```diff
@@ -66,7 +66,12 @@ def get_shared_cache_location(self):
         The shared location is a directory which shares data from
         the image buildsystem host with the image root system.
         """
-        return 'var/cache/kiwi'
+        from .cli import Cli
+        global_args = Cli().get_global_args()
+        if global_args['--shared-cache-dir']:
+            return global_args['--shared-cache-dir']
+        else:
+            return 'var/cache/kiwi'
```

Comment:
which reminds me that I need to care for this when the user specifies the location